docs/use-cases/deploy-ml.md

```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
 linkTitle: "Train and deploy classification models"
 weight: 50
 type: "docs"
-layout: "empty"
-canonical: "ml/"
+tags: ["data management", "data", "services"]
+no_list: true
+description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
+image: "/ml/training.png"
+imageAlt: "Machine Learning"
+images: ["/ml/training.png"]
 ---
+
+Deploy an image classification model onto your machine with Viam's machine learning (ML) service.
```

Comment:
Maybe? I think if it just covers deploying that's not covering the full thing but if it covers more it's not just the service
```suggestion
Create and deploy an image classification model onto your machine with Viam's machine learning (ML) capabilities.
```